src/commands/dev/dev.js

```diff
@@ -366,6 +366,19 @@ const validateShortFlagArgs = (args) => {
   return args
 }
 
+const validateGeoArgs = (arg) => {
+  if (!['cache', 'mock', 'update'].includes(arg) && arg.length !== 2) {
```

Comment:
What is the `arg.length` check doing?

Reply:
Validating that the code is two letters, so random input gives the helper message

Reply:
Ah, I see! Mind adding a comment to make that more explicit?

Reply:
Updated to use a separate CLI flag, so closing out this thread